Makefile

```diff
@@ -311,9 +311,15 @@ build-status:
 test-unit: buildenv-dirs "$(KUSTOMIZE)"
 	@./scripts/test-unit.sh $(NOMOS_GO_PKG)
 
+# small unit test to verify the behavior of an example kustomization for manual install
+.PHONY: test-kustomization
+test-kustomization:
+	$(MAKE) build-manifests STAGING_DIR=$(OUTPUT_DIR)/testing REGISTRY=gcr.io/cs-test IMAGE_TAG=placeholder
```

Comment:
This overwrites the previously defined `STAGING_DIR`. Is it intentional?

Reply:
Yes this is intentional

Reply:
Why?

Reply:
It generates manifests (without building/publishing images) for use with the small kustomize unit test that is added in this PR. The unit test uses manifests with static image tags so that the golden file doesn't drift with every commit. It's written to a different directory than the usual one so that it doesn't clobber real manifests that are written to the staging dir